test/033_wsteth.ts

```diff
@@ -0,0 +1,248 @@
+    )) as ChainlinkAggregatorV3Mock
+    await stethEthAggregator.set('992966330000000000')
+    await chainlinkMultiOracle.setSource(STETH, steth.address, ETH, weth.address, stethEthAggregator.address)
+
+    // Setting up stethUsd Chainlink oracle
+    stethUsdAggregator = (await deployContract(
+      ownerAcc,
+      ChainlinkAggregatorV3MockArtifact
+    )) as ChainlinkAggregatorV3Mock
+    await stethUsdAggregator.set('419348926478')
+    await chainlinkMultiOracle.setSource(STETH, steth.address, USDC, usdc.address, stethUsdAggregator.address)
+
+    // Setting up composite multi oracle
+    compositeMultiOracle = (await deployContract(ownerAcc, CompositeMultiOracleArtifact)) as CompositeMultiOracle
+    compositeMultiOracle.grantRoles(
+      [
+        id(compositeMultiOracle.interface, 'setSource(bytes6,bytes6,address)'),
+        id(compositeMultiOracle.interface, 'setPath(bytes6,bytes6,bytes6[])'),
+      ],
+      owner
+    )
+
+    // Set up the CompositeMultiOracle to draw from the ChainlinkMultiOracle
+    await compositeMultiOracle.setSource(WSTETH, STETH, lidoOracle.address)
+    await compositeMultiOracle.setSource(STETH, ETH, chainlinkMultiOracle.address)
+    await compositeMultiOracle.setSource(STETH, USDC, chainlinkMultiOracle.address)
+
+    // Set path for wsteth-steth-eth
+    await compositeMultiOracle.setPath(WSTETH, ETH, [STETH])
+
+    // Set path for wsteth-steth-USDC
+    await compositeMultiOracle.setPath(WSTETH, USDC, [STETH])
+
+    cTokenMultiOracle = (await deployContract(ownerAcc, CTokenMultiOracleArtifact)) as CTokenMultiOracle
+    await cTokenMultiOracle.grantRole(id(cTokenMultiOracle.interface, 'setSource(bytes6,bytes6,address)'), owner)
+    await cTokenMultiOracle.setSource(cUSDCId, USDC, cUSDC.address)
+    await cUSDC.set(WAD.mul(2).div(100)) // USDC has 6 + 10 decimals
+  })
+
+  it('WstETH could be added as an asset', async () => {
+    await wand.addAsset(WSTETH, WSTETHADD)
+  })
+
+  it('Convert WstETH into an ilk', async () => {
+    await witch.setIlk(WSTETH, 4 * 60 * 60, WAD.div(2), 1000000, 0, 18)
+    await wand.makeIlk(baseId, WSTETH, lidoOracle.address, 1000000, 1000000, 1000000, 6)
+  })
+
+  it('Build & test ETH Borrowing', async () => {
+    const { timestamp } = await ethers.provider.getBlock('latest')
+    const THREEMONTHS = timestamp + 3 * 31 * 24 * 60 * 60
+
+    // Add a series
+    await wand.addSeries(seriesId, baseId, THREEMONTHS, [WSTETH], seriesId, seriesId)
+
+    const fyToken = (await ethers.getContractAt(
+      'FYToken',
+      (await cauldron.series(seriesId)).fyToken,
+      ownerAcc
+    )) as FYToken
+
+    const pool = (await ethers.getContractAt('PoolMock', await env.ladle.pools(seriesId), ownerAcc)) as PoolMock
+    await pool.mint(ownerAcc.address, true, 0)
+    await fyToken.grantRole(id(fyToken.interface, 'mint(address,uint256)'), ownerAcc.address) // Only test environment
+    await fyToken.mint(pool.address, WAD.mul(2100000))
+    await pool.sync()
```

Comment:
You can skip the pool initialization if you use a series that was deployed by `fixtures`